pangeo_forge_orchestrator/models.py

```diff
@@ -1,10 +1,54 @@
 from datetime import datetime
 from enum import Enum
-from typing import Optional
+from typing import List, Optional
 
-from sqlmodel import SQLModel
+from sqlmodel import Field, SQLModel
 
-from .model_builders import MultipleModels
+from .model_builders import MultipleModels, RelationBuilder
+
+# Bakery --------------------------------------------------------------------------------
+
+
+class BakeryBase(SQLModel):
+    """Information about a Pangeo Forge Bakery.
+
+    :param region: The cloud region.
+    :param name: The bakery name.
+    :param description: A description of this bakery.
+    """
+
+    region: str  # TODO: Categorical constraint.
+    name: str  # TODO: Unique constraint.
+    description: str
+
+
+class BakeryRead(BakeryBase):
+    """The bakery read model. See ``RecipeRunRead`` docstring in this module for further detail.
+    """
+
+    id: int
+
+
+# Feedstock -----------------------------------------------------------------------------
+
+
+class FeedstockBase(SQLModel):
+    """Information about a Pangeo Forge Feedstock.
+
+    :param github_repo: The url of the associated GitHub repo.
+    """
+
+    github_repo: str  # TODO: Pydantic URL constraint.
```

Comment:
Pros and cons of this vs
```suggestion
    github_org: str   # org name, e.g pangeo-forge
    github_repo: str  # just the repo name e.g. staged-recipes
```
?

Reply:
I like the conciseness of passing just a repo name. (Especially assuming we can default `github_org="pangeo-forge"`.)

But in response JSON, it would be nice to have a copy-and-pastable URL? If we go with this suggestion, I guess we could generate the full URL as a @property somewhere.